odpf/entropy/v1beta1/service.proto

```diff
@@ -166,3 +171,20 @@ message GetLogRequest {
 message GetLogResponse {
   LogChunk chunk = 1;
 }
+
+message ResourceHistory {
```

Comment:
this naming still feels wrong to me since this message itself is not the history but multiple entries of this message together form the history. 

could we name it as `ResourceSpecRevision` or something like that?

Reply:
We can call it `revision`, that works too.

Reply:
Don't want to add `Spec` to the name so it still remains a little generic in case we want to include other fields in future.

Reply:
I added `Spec` because we are only maintaining history of changes to the spec. but I am okay either way.